protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/MoveLiquidTools/LiquidClassesStepTools.tsx

```diff
@@ -8,25 +8,45 @@ import {
   SPACING,
   StyledText,
 } from '@opentrons/components'
-import { getSortedLiquidClassDefs } from '@opentrons/shared-data'
+import {
+  PIPETTE_NAMES_MAP,
+  getDisableLiquidClasses,
+  getSortedLiquidClassDefs,
+} from '@opentrons/shared-data'
 import { getLiquidEntities } from '../../../../../../step-forms/selectors'
 import { getLiquidClassDisplayName } from '../../../../../../liquid-defs/utils'
-
+import { selectors as stepFormSelectors } from '../../../../../../step-forms'
 import type { ChangeEvent, Dispatch, SetStateAction } from 'react'
+import type { LiquidClassesOption } from '@opentrons/shared-data'
+import type { FormData } from '../../../../../../form-types'
 import type { FieldPropsByName } from '../../types'
 
 interface LiquidClassesStepToolsProps {
   propsForFields: FieldPropsByName
+  formData: FormData
   setShowFormErrors?: Dispatch<SetStateAction<boolean>>
 }
 export const LiquidClassesStepTools = ({
   propsForFields,
+  formData,
   setShowFormErrors,
 }: LiquidClassesStepToolsProps): JSX.Element => {
   const { t } = useTranslation(['liquids'])
   const liquids = useSelector(getLiquidEntities)
+  const pipetteEntities = useSelector(stepFormSelectors.getPipetteEntities)
   const sortedLiquidClassDefs = getSortedLiquidClassDefs()
-
+  const pipetteName = pipetteEntities[formData.pipette].name
```

Comment:
not practically possible to reach, but let's add a null check to avoid a whitescreen
```suggestion
  const pipetteName = pipetteEntities[formData.pipette]?.name
```